### PR TITLE
Enhance user management in dev portal

### PR DESCRIPTION
**Pull request by NipuniBhagya**

### CODE REVIEW

File: apps/admin-portal/src/components/users/add-user.tsx [hunk 1]
```diff
@@ -50,14 +51,86 @@ export const AddUser: React.FunctionComponent<AddUserProps> = (props: AddUserPro
...
+     * @param values
+     */
+    const handleUserNameChange = (values) => {
+        setUpdatedUsername(values?.get("userName")?.toString());
+    };
+
+    const generateRandomPassword = () => {
+        setRandomPassword(generate({ length: 10, numbers: true }));
+    };
+
```
Comment (DimalChandrasiri) [inline apps/admin-portal/src/components/users/add-user.tsx:133]: Function parameters and return types should be added.

File: modules/react-components/src/transfer/tranfer.tsx [hunk 2]
```diff
@@ -67,7 +67,9 @@ export const TransferComponent: FunctionComponent<PropsWithChildren<TransferComp
                                                         handleListSearch={ handleUnelectedListSearch }
                                                         placeholder={ searchPlaceholder }
                                                     />
+                                                    <Segment className="transfer-list-segment">
                                                     { list }
```
Comment (DimalChandrasiri) [inline modules/react-components/src/transfer/tranfer.tsx:71]: Should have an indentation for the child `{ list }` element for readability.

File: apps/admin-portal/src/components/users/add-user.tsx (hunk 1 above)
Comment (NipuniBhagya) [reply apps/admin-portal/src/components/users/add-user.tsx:137]: Resolved with e7e4251

File: modules/react-components/src/transfer/tranfer.tsx (hunk 2 above)
Comment (NipuniBhagya) [reply modules/react-components/src/transfer/tranfer.tsx]: Resolved with e7e4251

**Pull request merged**